### DISCUSSION TITLE
Can Emgu's VideoCapture determine the connection timeout?

### DISCUSSION THREAD

**Discussion by BeautyOfCoding** [emgucv/emgucv#825] [Q&A]
I use VideoCapture to read the RTSP stream of a webcam, but if the fileName error or doesn't exist, for example:
`VideoCapture  _capture = new  new Capture("rtsp://admin:123456@192.168.1.198:554/Streaming/Channels/301");`
In fact, the RTSP service corresponding to this fileName is not enabled, or there is no such IP in the LAN, at this time you will find that this line of code will always be blocked here, this is a very bad experience, does Emgu provide an API for relevant timeout judgment?

**Comment by leitengcn** [comment]
Have you found a solution to the same problem

**Reply by BeautyOfCoding** [reply]
No
